<?xml version='1.0' ?>

<launch>
  <arg name="use_sim_time" default="false"/>
  <arg name="failover_mode" default="false"/>

  <!-- Common launch -->
  <include file="$(find-pkg-share rmf_demos)/common.launch.xml">
    <arg name="use_sim_time" value="$(var use_sim_time)"/>
    <arg name="viz_config_file" value ="$(find-pkg-share rmf_demos)/include/office/office.rviz"/>
    <arg name="config_file" value="$(find-pkg-share om_fleet)/loglab.building.yaml"/>
    <arg name="dashboard_config_file" value="$(find-pkg-share om_fleet)/dashboard_config.json"/>
  </include>

  <group>
    <include file="$(find-pkg-share om_fleet)/fleet_manager.launch.xml">
      <arg name="config_file" value="$(find-pkg-share om_fleet)/config.yaml"/>
      <arg name="nav_graph_file" value="$(find-pkg-share om_fleet)/0.yaml" />
    </include>
  </group>

</launch>
